ModularTegustation/tegu_items/workshop/templates/basic.dm

```diff
@@ -186,7 +186,7 @@ Katana - Use in hand to dash
 		return
 	. = ..()
 	var/atom/throw_target = get_edge_target_turf(target, user.dir)
-	if(!target.anchored)
+	if(target && !target.anchored)
```

Comment:
```suggestion
	if(target && !target?.anchored)
```
adding a `?` after a variable tells the game it could be a null, this may resolve runtimes where it tries to check for `if(null && !null.anchored)`